python/pyspark/sql/functions.py

```diff
@@ -33,17 +33,23 @@
 __all__ = ['countDistinct', 'approxCountDistinct', 'udf']
 
 
-def _create_function(name, doc=""):
+def _function_obj(sc, is_math=False):
+    if not is_math:
+        return sc._jvm.functions
+    else:
+        return sc._jvm.mathfunctions
+
+
+def _create_function(name, doc="", is_math=False):
```

Comment:
how about changing is_math to "jvm_class" ?

and then remove _function_obj, and just pass `sc._jvm.functions` or `sc._jvm.mathfunctions` in.